<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindBar</class>
 <widget class="QDialog" name="FindBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="find_mod">
       <item>
        <property name="text">
         <string>Has</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>StartWith</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EndsWith</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Regex</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_find">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="filter_node">
       <property name="text">
        <string>Node</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filter_io">
       <property name="text">
        <string>GraphIO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filter_var">
       <property name="text">
        <string>Variable</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="ret_list"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
